requirements.txt

```diff
@@ -3,5 +3,5 @@ databases[postgresql]
 python-dateutil
 SQLAlchemy
```

Comment:
Should we pin the version of sqlalchemy too, just in case? This is probably not the version that the other services are using though so if we pin it make sure we pin it in all requirements.txt otherwise something bad probably happens...

Reply:
Good idea, I'm going to pin it